src/main/java/catan/domain/model/game/Resources.java

```diff
@@ -75,6 +76,38 @@ public void setStone(int stone) {
         this.stone = stone;
     }
 
+    public int getResource(HexType resource) {
+        switch (resource) {
+            case BRICK:
+                return getBrick();
+            case WOOD:
+                return getWood();
+            case SHEEP:
+                return getSheep();
+            case STONE:
+                return getStone();
+            case WHEAT:
+                return getWheat();
+            default:
+                return 0;
```

Comment:
if I pass some wrong HexType that is not listed in the switch-case, should I get 0? I think it is incorrect